pyproject.toml

```diff
@@ -62,6 +62,7 @@ lint = [
   "pre-commit>=2.20.0",
 ]
 test = [
+  "bioio-czi @ git+https://github.com/bioio-devs/bioio-czi.git@main",
```

Comment:
Is this supposed to point to the repo for integration?

Reply:
I imagine if we want to test against a plugin repo, we'd rather test against `main` than the latest release. This gives us more flexibility, like when coordinating ~simultaneous changes in both repos.

Reply:
Ok this works for me then. Initially I would think that we would only want to run it against "approved/passing" releases but I can see the utility here.